lib/ain-evm/src/core.rs

```diff
@@ -337,32 +336,24 @@ impl EVMCoreService {
     /// Result cannot be used safety unless cs_main lock is taken on C++ side
     /// across all usages. Note: To be replaced with a proper lock flow later.
     ///
-    pub unsafe fn validate_raw_tx(
-        &self,
-        tx: &str,
-        queue_id: u64,
-        pre_validate: bool,
-        block_fee: U256,
-    ) -> Result<ValidateTxInfo> {
+    pub unsafe fn prevalidate_raw_tx(&self, tx: &str, queue_id: u64) -> Result<ValidateTxInfo> {
```

Comment:
Instead of changing the naming convention, can we just call this validate_raw_tx to maintain consistency with the validate_transferdomain_raw_tx?

Reply:
Resolved in [95c4c7f](https://github.com/DeFiCh/ain/pull/2533/commits/95c4c7f0c7ab57f42974defbc87c3a49082f7437)